client/src/utils.ts

```diff
@@ -68,10 +70,26 @@ export function switchTheme(theme: string, set: (theme: string) => void) {
   }
 }
 
+export function isPayingSubscriber(user) {
```

Comment:
`isPlusSubscriber()` might be a more diplomatic name, in which case we could also check that the `user.subscriptionType` contains `plus`.

Reply:
So, a free account will not have a `subscriptionType` of `plus`?

Reply:
That's my understanding:

```ts
export enum SubscriptionType {
  MDN_CORE = "",
  MDN_PLUS_5M = "mdn_plus_5m",
  MDN_PLUS_5Y = "mdn_plus_5y",
  MDN_PLUS_10M = "mdn_plus_10m",
  MDN_PLUS_10Y = "mdn_plus_10y",
}
```

Only Plus subscription types start with `mdn_plus`. We could also check for that.

Reply:
Free accounts don't have a subscription, that's why their subscriptionType is `""`.